types/buffer.d.ts

```diff
@@ -1,12 +1,9 @@
 declare module "buffer" {
   export type BufferEncoding =
-    | "utf8"
     | "utf-8"
-    | "utf16le"
     | "utf-16le"
+    | "windows-1252"
     | "base64"
-    | "iso88591"
-    | "iso-8859-1"
     | "hex";
```

Comment:
We might want to keep the typing for the variations

Reply:
It is not accurate to leave only some aliases. We now enumerate all labels that can be mapped.